extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -235,7 +235,7 @@ export class RealTimeConfigManager {
     }
 
     const isGloballyValid = this.isValidCalloutForConsentState(this.rtcConfig_);
-    this.rtcConfig_.urls = this.rtcConfig_.urls.filter(
+    this.rtcConfig_.urls = (this.rtcConfig_.urls || []).filter(
```

Comment:
urls is undefined/null, this means you'll be allocating it.  Just want to make sure that is your intent as opposed to just wrapping in an if

Reply:
yep